build.sbt

```diff
@@ -277,6 +286,19 @@ lazy val `AbstractInterpretationFramework` = (project in file("OPAL/ai"))
   .dependsOn(br % "it->it;it->test;test->test;compile->compile")
   .configs(IntegrationTest)
 
+lazy val ifds = `IFDS`
```

Comment:
As we're adding sub-projects here, please also update:
- Readme.md
- OPAL/ProjectDependencies.{graffle,svg}
- src/site/UsingOPAL.md
- maybe more?

Reply:
Is there another Tool that can edit graffle files?